src/ffi.rs

```diff
@@ -34,7 +33,7 @@ pub struct Position {
 // convert a Polylabel result Point into values that can be sent across the FFI boundary
 impl<T> From<Point<T>> for Position
 where
-    T: Float + Signed,
+    T: GeoFloat,
```

Comment:
I also got rid of the `Signed` - GeoFloat currently doesn't inherit from `Signed`, so it wasn't redundant, but as far as I can tell it's not (or no longer) necessary.